flink-runtime/src/main/java/org/apache/flink/runtime/state/ttl/TtlStateSnapshotTransformer.java

```diff
@@ -55,9 +54,9 @@ boolean expired(long ts) {
 	}
 
 	private static long deserializeTs(
-		byte[] value, int offset) throws IOException {
+		byte[] value) throws IOException {
 		return LongSerializer.INSTANCE.deserialize(
-			new DataInputViewStreamWrapper(new ByteArrayInputStream(value, offset, Long.BYTES)));
+			new DataInputViewStreamWrapper(new ByteArrayInputStream(value, 0, Long.BYTES)));
```

Comment:
You could use a field with the new `ByteArrayDataInputView` to avoid creating the wrapped stream on each deserialize.